qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -237,7 +253,23 @@
           "project_url"
         ]
       },
-      "buttons": [],
+      "buttons": [
+        {
+          "enabled": true,
+          "type": "manifest",
+          "title": "Download Manifest",
+          "leftIcon": "datafile",
+          "rightIcon": "download",
+          "fileName": "manifest.json"
+        },
+        {
+          "enabled": true,
+          "type": "export-to-workspace",
+          "title": "Export File Manifest to Workspace",
```

Comment:
the only tab that can have the "export to workspace" button is the Genomics tab. The other ones don't contain GUIDs